configs/MSI/Z170A-Gaming-Pro.conf

```diff
@@ -0,0 +1,42 @@
+chip "nct6793-*"
+
+    label in0 "Vcore"
+    
+    label in2 "AVSB"
+    set in2_min  3.3 * 0.90
+    set in2_max  3.3 * 1.10
+    
+    label in3 "3VCC"
+    set in3_min  3.3 * 0.90
+    set in3_max  3.3 * 1.10
+    
+    label in7 "3VSB"
+    set in7_min  3.3 * 0.90
+    set in7_max  3.3 * 1.10
+    
+    label in8 "Vbat"
+    set in8_min  3.0 * 0.90
+    set in8_max  3.0 * 1.10
+    
+    label in9 "Vtt"
+    set in9_min  1.0 * 0.90
+    set in9_max  1.0 * 1.10
+
+    set in1_max 2.048
+    set in4_max 2.048
+    set in5_max 2.048
+    set in6_max 2.048
+    set in10_max 2.048
+    set in11_max 2.048
+    set in12_max 2.048
+    set in13_max 2.048
+    set in14_max 2.048
+    
+    ignore temp2
+    ignore temp3
+    ignore temp4
+    ignore temp5
+    ignore temp6
+    ignore temp8
+    ignore temp9
+    ignore temp10
```

Comment:
Please add a comment explaining why the temperatures are ignored.

Reply:
These temperature inputs don't report anything for this particular motherboard. That's why they're ignored. I'll add a comment explaining why.

Reply:
Well surely they report *something*, otherwise you wouldn't be ignoring them :).  Do they report something useful? I guess not. But what are the values you're seeing - e.g. always zero, fluctuating values, nonsensical values? That's what I'd like to see in that comment.

Reply:
For one input, I saw consistent 17 degrees celsius (not possible since I live in an area with hot climate with ambient temperature around 30+ degrees celsius). Some others were zero, or negative. They were clearly nonsense values, and there was no way to hook up external sensors to them, as far as I could tell. Hence why they're ignored.

Reply:
OK, thanks! Please put that in the config file.